### DISCUSSION TITLE
Hello friend, I'm Brazilian, and I used your script to do live, but it started to cause a problem when I click on making the chat transparent, sorry I can't make a donation in terms of thanks, but if you can fix it, it's my idea of stream works, I swear I'll leave my donation here for you

### DISCUSSION THREAD

**Discussion by harveyibr** [baffler/Transparent-Twitch-Chat-Overlay#259] [Q&A]
This is the error it displays

![image](https://github.com/baffler/Transparent-Twitch-Chat-Overlay/assets/163459878/a45f9473-f357-423d-8135-49fd2f7ce0fa)

**Comment by baffler** [comment]
Can you try the latest release and see if it fixes it? I couldn't reproduce this error myself but I made some changes that might fix the issue.
